internal/kafka/command_quota_create.go

```diff
@@ -13,23 +13,22 @@ import (
 
 func (c *quotaCommand) newCreateCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create <name>",
 		Short: "Create a Kafka client quota.",
-		Args:  cobra.NoArgs,
+		Args:  cobra.ExactArgs(1),
 		RunE:  c.create,
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Create client quotas for service accounts "sa-1234" and "sa-5678" on cluster "lkc-1234".`,
-				Code: `confluent kafka quota create --name clientQuota --ingress 500 --egress 100 --principals sa-1234,sa-5678 --cluster lkc-1234`,
+				Code: `confluent kafka quota create clientQuota --ingress 500 --egress 100 --principals sa-1234,sa-5678 --cluster lkc-1234`,
```

Comment:
It would be great to use lower-kebab-case for all of our args / flag values / etc.
```suggestion
				Code: `confluent kafka quota create my-client-quota --ingress 500 --egress 100 --principals sa-1234,sa-5678 --cluster lkc-1234`,
```